url_shortener/urls/templates/index.html

```diff
@@ -0,0 +1,26 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Url Shortener</title>
+</head>
+<body>
+    <main>
+        <h1>Url Shortener</h1>
+
+        <form method="POST">
```

Comment:
removing the 'hidden' attribute in browser dev tools allowed the input to be displayed, but not sure how you could change this from a django perspective.